src/reducers/index.js

```diff
@@ -105,102 +105,139 @@ function getInitialState() {
   };
 }
 
-function reducers(state = getInitialState(), action) {
-  switch (action.type) {
-    case ActionTypes.INITIALIZE: {
-      const { props } = action;
-
-      let newState = setStateFromProps(state, props);
-      newState = initializeRowHeightsAndOffsets(newState);
-      const scrollAnchor = getScrollAnchor(newState, props);
-      newState = computeRenderedRows(newState, scrollAnchor);
-      return columnStateHelper.initialize(newState, props, {});
-    }
-    case ActionTypes.PROP_CHANGE: {
-      const { newProps, oldProps } = action;
-      let newState = setStateFromProps(state, newProps);
-
-      if (oldProps.rowsCount !== newProps.rowsCount ||
-        oldProps.rowHeight !== newProps.rowHeight ||
-        oldProps.subRowHeight !== newProps.subRowHeight) {
+const reducerSlice = createSlice({
+  name: 'Reducer',
+  initialState: getInitialState(),
+  reducers: {
+    initialize: {
+      reducer(state, action) {
+        const { props } = action.payload;
+        let newState = setStateFromProps(state, props);
         newState = initializeRowHeightsAndOffsets(newState);
+        const scrollAnchor = getScrollAnchor(newState, props);
+        newState = computeRenderedRows(newState, scrollAnchor);
+        return columnStateHelper.initialize(newState, props, {});
+      },
+      prepare(props) {
```

Comment:
What's the point of having `prepare` here ? Seems like it doesn't provide any benefit.